homeassistant/components/airtouch4/strings.json

```diff
@@ -12,6 +12,9 @@
         "title": "Set up your AirTouch 4 connection details.",
         "data": {
           "host": "[%key:common::config_flow::data::host%]"
+        },
+        "data_description": {
+          "host": "The hostname or IP address of the device running your AirTouch controller."
```

Comment:
This is a device? Should it be:

"The hostname or IP address of your AirTouch controller."